modules/packages/BLAS.chpl

```diff
@@ -621,11 +634,11 @@ module BLAS {
 
     Solves the matrix equation::
 
-      op(A) * X = alpha * B
+      op(A) * X := alpha * B
```

Comment:
No := for trsm, just fro trmm (I had incorrectly taken that off the list previously)